public/app/features/alerting/unified/home/GettingStarted.tsx

```diff
@@ -182,26 +146,25 @@ const getWelcomeHeaderStyles = (theme: GrafanaTheme2) => ({
     color: theme.colors.text.secondary,
     paddingBottom: theme.spacing(2),
   }),
-  ctaContainer: css`
-    padding: ${theme.spacing(4, 2)};
-    display: flex;
-    gap: ${theme.spacing(4)};
-    justify-content: space-between;
-    flex-wrap: wrap;
-
-    ${theme.breakpoints.down('lg')} {
-      flex-direction: column;
-    }
-  `,
-
-  separator: css`
-    width: 1px;
-    background-color: ${theme.colors.border.medium};
+  ctaContainer: css({
+    padding: theme.spacing(2),
```

Comment:
I changed the padding here slightly so that the welcome banner ("Alert rules, Contact points...") and "How it works" were both left aligned with the same spacing. If this was intentional originally then happy to revert :)